data/human/jobs.txt

```diff
@@ -3903,7 +3903,7 @@ mission "Rescue Slaves [0]"
 	name "Help slaves escape to <planet>"
 	job
 	repeat
-	description "Slaves with valuable skills are asking for rescue. Pick up all <bunks> of them on <stopovers> and take them to them <destination> for <payment>. Expect bounty hunters."
+	description "An anonymous group will pay <payment> to transport <bunks> escaped slaves from <stopovers> to <planet>. You will have a passcode for landing clearance. Expect bounty hunters."
```

Comment:
Nice.

If I may make one more suggestion, it would be changing "escaped slaves" to something like "passengers escaping slavery"; emphasizing their personhood rather than their former conditions.

Reply:
Do you like this description for the New Tortuga jobs?

![escaped-slaves-try5](https://github.com/endless-sky/endless-sky/assets/116329264/8f6d9a80-a2c6-4abe-9e92-d32598d5a7b9)

The job boards on pirate planets still describe them as slaves, because that is how they're viewed on pirate planets.

It does look a little odd when they're being emancipated from Freedom, but that's unavoidable.

Reply:
It sparks a great deal of joy, I like it a lot; I hope we get more emancipation content in the future.